packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -35,6 +35,12 @@ const TRANSITION_TIME = 200; // Keep in sync with dso-viewer-grid.variables.scss
   shadow: true,
 })
 export class ViewerGrid {
+  private sizeLabelMap = new Map<MainSize, string>([
```

Comment:
Met het gebruik van Map is natuurlijk niets mis, maar het nadeel daarvan is dat het maar weinig afdwingt over de inhoud.

Ik heb even wat gespeeld met de TypeScript playground:

![image](https://user-images.githubusercontent.com/9696261/188440405-d618e83a-9202-4c02-9550-de31055c3c65.png)

Kijk maar of je het de moeite vindt.